src/tactic/ui/tools/pipeline_wdg.py

```diff
@@ -7594,6 +7660,13 @@ def get_display(self):
             if (bvr.uncategorized) {
                 var row = bvr.src_el.getParent(".spt_table_row_item");
                 row.setAttribute("spt_dynamic", true);
+
+                var tuple = spt.table.get_child_rows(row);
```

Comment:
Can you delete this code? I've changed the way export so that it simply ignores all children below dynamic.
This caused a bug when dragging uncategorized workflows into a category... they still had the dynamic flag, so they were ignored.